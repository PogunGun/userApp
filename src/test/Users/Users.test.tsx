import { MemoryRouter } from 'react-router-dom';
import UserCard from '../../components/User/userCard';
import { renderWithProviders } from '../utils/test-utils';

describe('UserCard', () => {
  const store = {};
  test('should Main render', () => {
    const { asFragment } = renderWithProviders(
            <MemoryRouter>
                <UserCard/>
            </MemoryRouter>,
            store,
    );

    expect(asFragment())
      .toMatchSnapshot();
  });
});
