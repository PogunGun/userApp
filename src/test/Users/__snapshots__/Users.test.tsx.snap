// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCard should Main render 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-j9wdxz"
  >
    <h1
      class="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
    >
      404
    </h1>
    <h6
      class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
    >
      User not Found
    </h6>
    <div
      aria-label="Disabled elevation buttons"
      class="MuiButtonGroup-root MuiButtonGroup-contained MuiButtonGroup-disableElevation css-uxd1ve-MuiButtonGroup-root"
      role="group"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContainedPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation css-mtnx2g-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Prev
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContainedPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation css-mtnx2g-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Next
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-1ntrnb6-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      Home
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
